roles/nas/tasks/main.yml

```diff
@@ -24,6 +24,14 @@
     loop_var: mount
   when: nas.mounts is defined
 
+- name: configure mirroring to secondary storage
+  cron:
+    name: "mirror NAS storages"
+    job: "rsync -ast --delete {{ nas.mount_base_dir }}/{{ nas.mounts.primary.name }} {{ nas.mount_base_dir }}/{{ nas.mounts.secondary.name }}"
+    special_time: weekly
+  become_user: "{{ nas.user }}"
+  when: (nas.mounts.primary is defined) and (nas.mounts.secondary is defined)
```

Comment:
primary should always be defined, check not required (will fail on usage if really not defined)